sdv/sequential/par.py

```diff
@@ -167,6 +167,23 @@ def preprocess(self, data):
         self.update_transformers(sequence_key_transformers)
         return super().preprocess(data)
 
+    def update_transformers(self, column_name_to_transformer):
```

Comment:
I'm only adding `update_transformers` because the other two methods are already implemented in base.

Reply:
I think this should be good. Can you double check that the correct transformers are assigned when calling `auto_assign_transformers`? For PAR, everything should be `None` except for datetimes